src/Microsoft.ApplicationInsights/DataContracts/ISupportRichPayloadEventSource.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.DataContracts
+{
+    using System.Diagnostics.Tracing;
+
+    /// <summary>
+    /// 
```

Comment:
Fill in the summary text. Something like: "Marker interface to indicate that a telemetry item supports serializing to an event source."